src/client/opamCommands.ml

```diff
@@ -532,9 +532,15 @@ let list ?(force_search=false) () =
       "Don't write anything in the output, exit with return code 0 if the list \
        is not empty, 1 otherwise."
   in
+  let no_depexts =
+    mk_flag ["no-depexts"]
+      "Disable external dependencies handling for the query. This can be used \
+       to list packages that are marked as unavailable because a not available \
+       system dependency."
```

Comment:
```suggestion
      "Disable external dependencies handling for the query. This can be used \
       to include packages that are marked as unavailable because of an unavailable \
       system dependency."
```
(with "list", at first I read it as, it lists _only_ those)